### PR TITLE
feat: condition time interval

### PR DESCRIPTION
**Pull request by Warumdk**
## Related issue
This does not solve an existing issue as far as I know.

## Proposed changes
I have added an extra condition that allows checking setting a time interval for a policy. To pre-address some questions: This could have been called number interval but I feel the imitate use case of time to be more obvious. I will change it if deemed necessary. Please let me know if this was already possible with the current implementation then I will close this PR.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

## Further comments

Thank you for your time!

### CODE REVIEW

File: docs/versioned_docs/version-v0.5/engines/acp-ory.md [hunk 1]
```diff
@@ -498,6 +498,54 @@ The following access request would be denied.
 }
 ```
 
+### Time Interval Condition
```
Comment (aeneasr) [inline docs/versioned_docs/version-v0.5/engines/acp-ory.md:501]: Can you please add these changes to docs/docs/engines/acp-ory.md as well? Thank you!

**Review by aeneasr — CHANGES_REQUESTED** [review]
Awesome 🎉 

Thank you for your contribution!

File: docs/versioned_docs/version-v0.5/engines/acp-ory.md (hunk 1 above)
Comment (Warumdk) [reply docs/versioned_docs/version-v0.5/engines/acp-ory.md:501]: Yeah no problem.

**Review by aeneasr — COMMENTED** [review]
Awesome, thank you for your contribution!

**Pull request merged**